snippets/ios/SampleProject/SampleProject/ViewController.swift

```diff
@@ -0,0 +1,417 @@
+//
+//  ViewController.swift
+//  SampleProject
+//
+//
+
+import UIKit
+import ThingIF
+
+class ViewController: UIViewController {
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        // Do any additional setup after loading the view, typically from a nib.
+
+        let owner = Owner(TypedID(.thing, id: "dummyID"), accessToken: "dummyToken")
```

Comment:
In this sample, no values are in `variables`
If I insert `owner`, it's `thingIFAPI(_ios)` like this,
```
let app=**
let owner=**
let api=ThingIFAPI(app, owner: owner)
```